examples/notebooks/beam-ml/rag_usecase/credentials.env

```diff
@@ -0,0 +1,20 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+OPENSEARCH_USERNAME="<ADD_YOUR>"
+OPENSEARCH_PASSWORD="<ADD_YOUR>"
```

Comment:
Rather than having these in a file outside of the notebook, could we write them as an executable notebook cell? So it would look like:

```
cell_str='''
OPENSEARCH_USERNAME="<REPLACE_WITH_YOUR_USERNAME>"
OPENSEARCH_PASSWORD="<REPLACE_WITH_YOUR_PASSWORD>"
'''

with open('credentials.env', 'w') as f:
  f.write(cell_str)
```

Reply:
Good point, we can save an additional file as part of repository and generate on the fly. Done.